runtime/lua/vim/lsp/completion.lua

```diff
@@ -596,7 +596,7 @@ local function on_complete_done()
 end
 
 --- @class vim.lsp.completion.BufferOpts
---- @field autotrigger? boolean Whether to trigger completion automatically. Default: false
+--- @field autotrigger? boolean  Default: false When true completion will only be triggered automatically if |v:char| is contained within the client's `triggerCharacters`.
```

Comment:
```suggestion
--- @field autotrigger? boolean  Default: false When true, completion triggers automatically based on the server's `triggerCharacters`.
```

Just a suggestion.

- Removes the "only" as it somehow implies that there's a way to have it auto-triggered in more cases.
- Uses `server` instead of `client` to make it clearer that the trigger characters are defined by the server. (Not sure about this, as technically we save the servers definition on the client - so this could be confusing?)

Reply:
let's go